test_scripts/WidgetSupport/Capabilities/008_GetSCs_no_app_specific_capabilities.lua

```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------------
+-- User story: SubtleAlert cases
+-- Use case: SubtleAlert
+-- Item: Happy path (UI only)
+--
+-- Requirement summary:
+-- [SubtleAlert] Capabilities: getting the displayCapabilities with subtleAlert related params
+--
+-- Description:
+-- Mobile application sends valid GetSystemCapability request with DISPLAYS systemCapabilityType 
+-- and gets SUCCESS resultCode only if SDL received OnSystemCapabilityUpdated notification from HMI
+-- with the displayCapabilities for specific appID
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+
+-- Steps:
+-- App requests GetSystemCapability with DISPLAYS systemCapabilityType
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if displayCapabilities is available for specified appID
+-- SDL responds with (resultCode: DATA_NOT_AVAILABLE, success:false) to mobile application
+-- HMI sends OnSystemCapabilityUpdated with the displayCapabilities for specific appID
+-- App requests GetSystemCapability with DISPLAYS systemCapabilityType one more time
+-- SDL responds with (resultCode: SUCCESS, success:true) and transfers the displayCapabilities to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/WidgetSupport/common')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local Functions ]]
+local function sendOnSCU()
+  local paramsToSDL = common.getOnSystemCapabilityParams()
+  paramsToSDL.appID = common.getHMIAppId()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnSystemCapabilityUpdated", paramsToSDL)
+  common.getMobileSession():ExpectNotification("OnSystemCapabilityUpdated", common.getOnSystemCapabilityParams())
+end
+
+local function sendGetSC(pSuccess, pResultCode)
+  local cid = common.getMobileSession():SendRPC("GetSystemCapability", { systemCapabilityType = "DISPLAYS" })
+  common.getMobileSession():ExpectResponse(cid, {
+    success = pSuccess,
+    resultCode = pResultCode 
+  })
+  :ValidIf(function(_, data)
+    if (true == data.payload.success) then
+      local params = common.getOnSystemCapabilityParams().systemCapability
+      if not common.isTableEqual(data.payload.systemCapability, params) then
+        return false, "\nExpected data:\n" .. common.tableToString(params, 1)
+          .. "\nActual data:\n" .. common.tableToString(data.payload.systemCapability, 1)
+      end
+    end
+    return true
+  end)
+end
```

Comment:
I propose to rework this function in a following way:
```
local function sendGetSC(pSuccess, pResultCode)
  local cid = common.getMobileSession():SendRPC("GetSystemCapability", { systemCapabilityType = "DISPLAYS" })
  local exp = nil
  if pResultCode == "SUCCESS" then exp = common.getOnSystemCapabilityParams().systemCapability end
  common.getMobileSession():ExpectResponse(cid, {
    success = pSuccess,
    resultCode = pResultCode,
    systemCapability = exp
  })
  :ValidIf(function(_, data)
      if data.payload.success == false and data.payload.systemCapability ~= nil then
        return false, "Struct 'systemCapability' is populated in erroneous response"
      end
      return true
    end)
end
```

In this version `systemCapability` would be verified for success and erroneous scenarios.